File: backend/src/main/java/com/pofo/backend/domain/resume/experience/dto/ExperienceRequest.java
package com.pofo.backend.domain.resume.experience.dto;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import java.time.LocalDate;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ExperienceRequest {
    @NotBlank
    private String name;
    @NotBlank
    private String department;
    @NotBlank
    private String position;
    @NotBlank
    private String responsibility;
    @NotNull
    private LocalDate startDate;
    @NotNull
    private LocalDate endDate;
}

File: backend/src/main/java/com/pofo/backend/domain/resume/experience/dto/ExperienceRequest.kt
package com.pofo.backend.domain.resume.experience.dto

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.NotNull
import java.time.LocalDate

data class ExperienceRequest(

    @field:NotBlank
    val name: String,

    @field:NotBlank
    val department: String,

    @field:NotBlank
    val position: String,

    @field:NotBlank
    val responsibility: String,

    @field:NotNull
    val startDate: LocalDate,

    @field:NotNull
    val endDate: LocalDate
)
